doc/write_your_own/components/Workflow1.ipynb

```diff
@@ -135,14 +96,21 @@
    "source": [
     "options = glob.glob('{}/*'.format(path.value)) \n",
     "\n",
-    "files = ipywidgets.SelectMultiple(\n",
+    "files = ipywidgets.Select(\n",
     "    description='Cube(list)',\n",
```

Comment:
How about 'Dataset(s)' instead? Technically what this widget will show is a list of datasets that could be loaded as cubes using Iris, rather than actually being cubes.